torch/_inductor/codegen/cpp_template_kernel.py

```diff
@@ -307,13 +360,28 @@ def store_output(
            c) If `src` is local, we need to add a local buffer for it and localize the `orig_src` buffer
               in `epilogue_nodes` with `src`.
         """
-        assert dst.get_size() == src.get_size(), f"{dst=}, {src=}"
+        # <TODO> Leslie: maybe split this function for Group GEMM is clearer since
+        # not much code sharing between this 2 path
```

Comment:
I have split the `store_output` into `store_output` and `store_outputs` in the following PR (the second one is for Grouped GEMM) since enabling epilogues fusion requires much more changes.